release-tag.sh

```diff
@@ -0,0 +1,7 @@
+version=`python setup.py --version`
+eval "$(ssh-agent -s)" #start the ssh agent
+chmod 400 ./openfisca_bot
```

Comment:
Without doing this, ssh-add will not add the key as it is not safe.
Is there a security problem with, during the build, decrypting the ssh key into a clear text file before adding it ?
The only risk I would see is that someone could, by mistake, print the ssh key - or the travis key that decrypts it - on the log by adding a wrong command to the release script or the travis configuration file. It doesn't sound really likely.

An alternative would be to try to use a github token, but anyway, if we want to do real continuous deployment in the future, we will need a ssh key.

@MattiSG ?